src/__tests__/__snapshots__/role-helpers.js.snap

```diff
@@ -1,15 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`logRoles calls console.log with output from prettyRoles 1`] = `
-"region:
-
-Name "":
-<section
-  data-testid="a-section"
-/>
-
---------------------------------------------------
```

Comment:
This change appears to be a result of the aria-query update. Maybe that's a bug?